models/event.js

```diff
@@ -245,10 +247,10 @@ Event.fromRecord = (record) => {
   }
   const event = new Event()
   event.event_id = decoded[Fields.event_id]
-  // Use collector timestamp as event creation time, forcing UTC
-  event.created_at = new Date(decoded[Fields.collector_tstamp] + '+0000')
```

Comment:
What will go in `created_at` with this line removed?

Reply:
By leaving it empty it will get set to the Date/Time the event is created in the database.